coalib/parsing/CliParsing.py

```diff
@@ -103,4 +103,15 @@ def check_conflicts(sections):
             ArgumentParser().error(
                 "'no_config' cannot be set together 'save' or 'find_config'.")
 
+        try:
+            if (
+                    not section.get('json', False) and
+                    (section.get('output', False) or
+                     section.get('relpath', False))):
+                ArgumentParser().error(
+                    "'output' or 'relpath' cannot be used without `--json`.")
+        except ValueError:
```

Comment:
`json, relpath` are boolean values, but `output` takes a string, and that causes a ValueError.